network-store-server/src/main/resources/db/changelog/changesets/changelog_2022-09-28T21:30:00Z.xml

```diff
@@ -0,0 +1,227 @@
+            <column name="ratiotapchangerloadtapchangingcapabilities2" type="BOOLEAN"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangertargetv2" type="FLOAT8"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerlowtapposition3" type="INTEGER"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangertapposition3" type="INTEGER"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerregulating3" type="BOOLEAN"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangertargetdeadband3" type="FLOAT8"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerterminalrefconnectableid3" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerterminalrefside3" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerregulationmode3" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerregulationvalue3" type="FLOAT8"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerlowtapposition3" type="INTEGER"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangertapposition3" type="INTEGER"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerregulating3" type="BOOLEAN"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangertargetdeadband3" type="FLOAT8"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerterminalrefconnectableid3" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerterminalrefside3" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerloadtapchangingcapabilities3" type="BOOLEAN"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangertargetv3" type="FLOAT8"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="cboutier" id="1664174319707-3">
+        <createTable tableName="tapchangerstep">
+            <column name="equipmentid" type="VARCHAR(255)"/>
```

Comment:
Shouldn't the columns of the PK be NOT NULL ?

Reply:
fixed